### ISSUE TITLE
Loading graphql.sql fails - relation "meta.pk"

### ISSUE THREAD

**Issue by aleak** [solidsnack/GraphpostgresQL#2]
Hi - thanks for making this public - quite excited to experiment, however I'm not getting past loading.

```
postgres=# \i graphql.sql 
BEGIN
psql:graphql.sql:3: NOTICE:  schema "graphql" does not exist, skipping
DROP SCHEMA
CREATE SCHEMA
SET
CREATE VIEW
CREATE VIEW
CREATE VIEW
CREATE FUNCTION
psql:graphql.sql:114: ERROR:  relation "meta.pk" does not exist
LINE 2:   SELECT cols FROM meta.pk WHERE meta.pk.tab = t;
```

**Comment by solidsnack** [comment]
I am not able to fix at present -- at a party -- but every reference to
`meta.` can be changed to `graphql.`.

**Comment by solidsnack** [comment]
Fixed in d144260
